ia/src/protocole.py

```
@@ -60,7 +62,8 @@ def handle_Dead(client, response) -> None:
     client.is_alive = False
     game.remove_client(client)
 
-def handle_fork(client, response) -> None:
+def handle_Fork(client, response) -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)